packages/application-components/src/experimental/components/product-picker-input/product-picker-input.spec.tsx

```diff
@@ -0,0 +1,216 @@
+import type { TRenderAppOptions } from '@commercetools-frontend/application-shell/test-utils';
+
+// import { graphql } from 'msw';
+// import { setupServer } from 'msw/node';
```

Comment:
Leftovers?

Reply:
yeah my bad. 
They are not supposed to be there :)